rotkehlchen/exchanges/coinbasepro.py

```diff
@@ -388,6 +388,14 @@ def query_online_deposits_withdrawals(
         account_to_currency = self.create_or_return_account_to_currency_map()
         for entry in raw_movements:
             try:
+                # Check if the transaction has been canceled. If so it should be skipped
```

Comment:
This is not just for canceled transactions. Just say "if the transaction has not been completed".

It's quite possible rotki will query coinbase pro, while a user withdrawal, or deposit is in flight.